source/Halibut.TestUtils.Contracts/CountingService.cs

```diff
@@ -31,5 +31,10 @@ public async Task<int> GetCurrentValueAsync(CancellationToken cancellationToken)
             await Task.CompletedTask;
             return service.GetCurrentValue();
         }
+        
+        public int CurrentValue()
```

Comment:
This makes testing easier, otherwise we need to await a mehtod that returns an int, on an expression we want to use ".Should().Be..." on.